src/View/State/ResetsState.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Statamic\View\State;
+
+interface ResetsState
+{
+    public function resetState();
```

Comment:
Please change this to:

```suggestion
    public static function resetStaticState();
```

I feel like it would be better to be more explicit that it's for resetting the static state.

A quick look through GitHub indicates `function resetState` is pretty common and this might conflict with someone already using the method.

You can leave the interface name the same.